### DISCUSSION TITLE
need help with new PAC and push

### DISCUSSION THREAD

**Discussion by Lee343W** [community/community#30535] [New to GitHub]
I try to push and and I can no longer enter pw so I use PAC as suggested by "https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls", but after I enter username + PAC I still get the same fatal: authentication failed. What am I missing?

**Comment by queenofcorgis** [comment]
Hi there!

I believe you are referring to a personal access token or a PAT. Please post the full command you used and its output. If you want to keep the repository name private you can replace it with ***, but the details are going to help us solve this 😄
